### PR TITLE
chore(uuid): Update package

### LINKED ISSUE #11: Request: Update to uuid module
Hi @nelsonic,

Thanks for this amazing little module 👍 

Can you guys update from legacy node-uuid to uuid?

It will be helpful to keep to the most current version.

### PR DESCRIPTION
**Pull request by goncaloneves**
Closes #11

@nelsonic let me know if you need anything more.

We are using this in production, so I will be keeping an eye on this one 😎

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -26,7 +26,7 @@
   },
   "homepage": "https://github.com/dwyl/aguid",
   "dependencies": {
-    "node-uuid": "^1.4.7"
+    "uuid": "^3.0.1"
```
Comment (nelsonic) [inline package.json]: thanks! 👍

**Comment by nelsonic** [comment]
@goncaloneves looks great. merging.

**Comment by nelsonic** [comment]
@goncaloneves the new/latest version on NPM `aguid@2.0.0` now uses `uuid` thanks to you!! 🚀 
Welcome to DWYL! ❤️  ✅  
Thank you for _contributing_ to our Open Source! 🎉

**Comment by goncaloneves** [comment]
@nelsonic Pleasure to contribute to @dwyl/ateam Open Source. ❤️   That is the beauty of tech in our days. Congrats on the organisation and cool contributions to the community.
Looks like we will be speaking sometime in the future. 🇬🇧  Keep it up until then!

**Comment by tunnckoCore** [comment]
Btw, just to mention to checkout the `cuid` package :p 
Seems very good and where I need such thing I use it. :)

**Comment by nelsonic** [comment]
@tunnckoCore great shout on https://www.npmjs.com/package/cuid 👍 
it _could_ be an option for us. thanks!

**Pull request merged**